From: Dean Nelson <dcn@sgi.com>
Subject: [PATCH] Define xp_partition_id and xp_region_size
References: bnc#442461


Define xp_partition_id and xp_region_size to their correct values.

Signed-off-by: Dean Nelson <dcn@sgi.com>
Acked-by: Bernhard Walle <bwalle@suse.de>

---

 drivers/misc/sgi-xp/xp_uv.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/misc/sgi-xp/xp_uv.c
+++ b/drivers/misc/sgi-xp/xp_uv.c
@@ -119,8 +119,8 @@ xp_init_uv(void)
 	BUG_ON(!is_uv());
 
 	xp_max_npartitions = XP_MAX_NPARTITIONS_UV;
-	xp_partition_id = 0;	/* !!! not correct value */
-	xp_region_size = 0;	/* !!! not correct value */
+	xp_partition_id = sn_partition_id;
+	xp_region_size = sn_region_size;
 
 	xp_pa = xp_pa_uv;
 	xp_remote_memcpy = xp_remote_memcpy_uv;
